ntia-algorithm.sigmf-ext.md

```diff
@@ -110,470 +189,110 @@ The `FrequencyDomainDetection` has the following properties:
-        "category" : "Outside. Coastal."
-      }
-    },
-    "ntia-location:coordinate_system" : {
-      "coordinate_system_type" : "CoordinateSystem",
-      "id" : "WGS_84",
-      "elevation_ref" : "MSL",
-      "elevation_unit" : "meter"
-    },
-    "ntia-sensor:calibration_datetime" : "2018-01-01T10:49:58.236Z",
-    "ntia-core:measurement" : {
-      "domain" : "Frequency",
-      "measurement_type" : "Scan",
-      "time_start" : "2018-01-01T07:59:42.792Z",
-      "time_stop" : "2018-01-01T08:00:37.792Z",
-      "frequency_tuned_low" : 3.45940625E9,
-      "frequency_tuned_high" : 3.65190625E9,
-      "frequency_tuned_step" : 1.925E7
-    }
-  },
-  "captures" : [ {
-    "core:sample_start" : 0,
-    "core:frequency" : 3.5501875E9,
-    "core:datetime" : "2018-01-01T07:59:42.792Z"
-  } ],
-  "annotations" : [ {
-    "ntia-core:annotation_type" : "FrequencyDomainDetection",
-    "core:sample_start" : 0,
-    "core:sample_count" : 458,
-    "core:comment" : "",
-    "ntia-algorithm:detector" : "fft_max_power",
-    "ntia-algorithm:number_of_ffts" : 10,
-    "ntia-algorithm:number_of_samples_in_fft" : 50,
-    "ntia-algorithm:window" : "Gauss-top",
-    "ntia-algorithm:equivalent_noise_bandwidth" : 962500.0,
-    "ntia-algorithm:frequency_start" : 3.45021875E9,
-    "ntia-algorithm:frequency_stop" : 3.65015625E9,
-    "ntia-algorithm:frequency_step" : 437500.0
-  }, {
-    "ntia-core:annotation_type" : "CalibrationAnnotation",
-    "core:sample_start" : 0,
-    "core:sample_count" : 458,
-    "core:comment" : " Calibration is done every 6 hours.",
-    "ntia-sensor:gain_preselector" : 27.241,
-    "ntia-sensor:noise_figure_sensor" : 7.638,
-    "ntia-sensor:enbw_sensor" : 962500.0000000001,
-    "ntia-sensor:mean_noise_power_sensor" : -94.28774890829693,
-    "ntia-sensor:temperature" : 14.611,
-    "ntia-sensor:mean_noise_power_units" : "dBm"
-  }, {
-    "ntia-core:annotation_type" : "SensorAnnotation",
-    "core:sample_start" : 0,
-    "core:sample_count" : 458,
-    "ntia-sensor:rf_path_index" : 0,
-    "ntia-sensor:overload" : false,
-    "ntia-sensor:attenuation_setting_sigan" : 3.0
-  } ]
-}
-```
-
-
-### 4.5  DigitalFilterAnnotation  Example
-```json
-{
-  "global": {
-	...
+    "core:version" : "1.0.0",
+    "core:num_channels" : 1,
+    "core:extensions" : [ {
+      "name" : "ntia-core",
+      "version" : "v2.0.0",
+      "optional" : false
+    }, {
+      "name" : "ntia-algorithm",
+      "version" : "v2.0.0",
+      "optional" : false
+    } ],
+    "ntia-core:classification" : "UNCLASSIFIED",
+    "ntia-algorithm:data_products" : [ {
+      "processing" : [ "psd_fft" ],
+      "name" : "power_spectral_density",
+      "description" : "Power spectral density with first and last 125 samples removed, computed from filtered data",
+      "series" : [ "max", "mean" ],
+      "length" : 625,
+      "reference" : "calibration terminal",
+      "xunits" : "Hz",
+      "yunits" : "dBm/Hz",
+      "xstep" : [ 16000.0 ],
```

Comment:
xunits, yunits, xstep, xstop, xstart should be removed in all of these data products (only x_units, x_start, x_stop, etc are defined)

Reply:
changed to x_units and y_units

Reply:
It looks like this caused x_units and y_units to be duplicated. Also I think xstep, xstop, and xstart need to be removed - x_step, x_stop, and x_start are already there. This also seems true for ystep, ystart, and ystop for one of the data products.